src/Components/PhotoUpload/Utils/fileUtils.ts

```diff
@@ -118,3 +121,34 @@ export const uploadPhoto = async (
     return
   }
 }
+
+export const uploadMyCollectionPhoto = async (
+  relayEnvironment: Environment,
+  photo: Photo,
+  updateProgress: (progress: number) => void,
+  acl: string = "private"
+) => {
+  const convectionKey = await getConvectionGeminiKey(relayEnvironment)
```

Comment:
**Q:** is there a reason why we are using convection gemini key here? maybe we can use something else

Reply:
I've copied the logic from the app. I'll have a look to see if we can actually get another key with a similar query ([getConvectionGeminiKey](https://github.com/artsy/force/blob/07842a95901dc88b2af88aa955f4e79434b0b404/src/Components/PhotoUpload/Mutations/getConvectionGeminiKey.ts#L4)).